cypress/e2e/views/common.view.ts

```diff
@@ -60,3 +60,4 @@ export const kebabMenuItem = "a.pf-c-dropdown__menu-item";
 export const commonTable = "table[aria-label='main-table']";
 export const dropdownClearSelection = "pf-v5-c-select__toggle-clear";
 export const footer = "footer";
+export const actionsButton = 'button[aria-label="Actions"]';
```

Comment:
Could we have a more meaningful name for this ?

Reply:
Solved in 90bcc68e960ea58a89cbfac10533e3a1c6966a55.